controllers/nginx/pkg/cmd/controller/nginx.go

```diff
@@ -252,6 +253,24 @@ func (n NGINXController) OnUpdate(cmap *api.ConfigMap, ingressCfg ingress.Config
 	}, n.testTemplate)
 }
 
+// Name returns the healthcheck name
+func (n NGINXController) Name() string {
+	return "Ingress Controller"
+}
+
+// Check returns if the nginx healthz endpoint is returning ok (status code 200)
+func (n NGINXController) Check(_ *http.Request) error {
+	res, err := http.Get("http://127.0.0.1:18080/healthz")
```

Comment:
are we exposing the output of a Check call on some standard port as a liveness or readiness check?